README.md

```diff
@@ -80,86 +80,17 @@ size = "small" | "medium" | "large" | [( "2" | "4" | "8" )] , "xlarge";
 
 # Series
 
-.                           |  C    |  CO   |  CX   |  GN   |  M    |  N
+.                           |  CX   |  GN   |  M    |  N    |  U    |  UO
 ----------------------------|-------|-------|-------|-------|-------|------
-*Has GPUs*                  |       |       |       |  ✓    |       |
-*Hugepages*                 |       |       |       |       |  ✓    |  ✓
-*Cache backed RAM*          |       |  ✓    |       |       |       |
-*Dedicated CPU performance* |       |       |  ✓    |       |       |
-*Burstable CPU performance* |  ✓    |  ✓    |       |  ✓    |  ✓    |  ✓
-*Isolated emulator threads* |       |       |  ✓    |       |       |
-*vNUMA*                     |       |       |  ✓    |       |       |
+*Has GPUs*                  |       |  ✓    |       |       |       |
+*Hugepages*                 |       |       |  ✓    |  ✓    |       |
+*Cache backed RAM*          |       |       |       |       |       |  ✓
+*Dedicated CPU performance* |  ✓    |       |       |       |       |
```

Comment:
Would `Dedicated CPU` reflect better the meaning than `Dedicated CPU performance`, because the performance is an implication from the  `dedicated`?

Reply:
Absolutely, thanks